tests/foreman/cli/test_syncplan.py

```diff
@@ -400,11 +397,9 @@ def test_negative_synchronize_custom_product_current_sync_date(self):
             )
             # validate the error message once unstubbed (#3611)
 
-    # This Bugzilla bug is private. It is impossible to fetch info about it.
-    @stubbed('Unstub when BZ1279539 is fixed')
     @tier4
     def test_positive_synchronize_custom_product_current_sync_date(self):
-        """Create a sync plan with current datetime as a sync date, add a
+        """Create a sync plan with a past datetime as a sync date, add a
```

Comment:
Current sync date (time) does not make sense on defining a SC for a simple reason. Creating takes time and one usually misses the trigger as creating takes time.
I would recommend getting rid of 'current' time test case and change it to the 'past' one.
Better test something than nothing.